autokeras/hypermodel/hyper_head.py

```diff
@@ -17,26 +17,28 @@ def build(self, hp, inputs=None):
 
 class ClassificationHead(HyperHead):
 
-    def __init__(self, **kwargs):
+    def __init__(self, binary=False, **kwargs):
```

Comment:
Shouldn't the number of classes be configurable here?

Reply:
If it is configurable, it would only be a security check for the targets user passed contains the same number of classes.